network-api/networkapi/wagtailpages/pagemodels/blog/blog_index.py

```diff
@@ -306,6 +313,68 @@ def extract_topic_information(self, topic_slug):
             'topic': topic_object
         }
 
+    def get_related_topics(self):
```

Comment:
Moved this topic up here because I thought it would make sense to have all the routes at the end of the file.